packages/agent-toolkit/src/monday-graphql/queries.graphql.ts

```diff
@@ -54,6 +54,21 @@ export const getBoardSchema = gql`
   }
 `;
 
+
+export const getBoardAllActivity = gql`
+  query GetBoardAllActivity($boardId: ID!, $fromDate: ISO8601DateTime!, $toDate: ISO8601DateTime!, $limit: Int = 1000, $page: Int = 1) {
```

Comment:
They might now have one now, but in general in other places we usually allow limit to be top 100 from what I recall

Reply:
I think 1000 is ok, also this is what Daniel H asked for. We can take it with him if you disagree